pkg/crc/machine/bundle/metadata.go

```diff
@@ -217,7 +217,7 @@ func (bundle *CrcBundleInfo) verify() error {
 		bundle.GetKernelPath(),
 		bundle.GetInitramfsPath(),
 	}
-	if bundle.IsOpenShift() {
+	if !bundle.IsPodman() {
```

Comment:
It seems the `verify()` method is not exactly doing what we expect. If we want an error if the metadata file does not contain data for `oc` or for a `kubeconfig` file, then we should remove `if file == "" { continue }` from the loop. 
Having both this check and changing the list of files which are checked according to the preset is odd.

Reply:
The commit log is still a bit misleading, with this change, we will get a complaint with the microshift bundle if its metadata specifies oc/kubeconfig files. But if the bundle metadata is missing oc/kubeconfig files, I don't think this will be caught.

Reply:
Hmm, I don't know why I did not see your message from 3 days ago before posting my message from today.